### DISCUSSION TITLE
Do you know the solution for this issue? ----> PT-PT vs PT-BR

### DISCUSSION THREAD

**Discussion by CROSSEO** [explosion/spaCy#6893] [Help: Best practices]
I just start using spacy in my models and I would like to know if it's possible to have the spacy.load("pt-pt") and not only spacy.load("pt").

The reason is that in portuguese you have **"pt-pt"** _that stands for Portuguese of Portugal_  and you have **"pt-br"** _that stands for portuguese of Brazil_. 

And when mixing this 2 languages  is like mixing en-uk and en-us, that you have for example: **British:** Grey _vs_ **American:** Gray.

Or 

Can anyone explain me a better solution to solve this challenge?

**Comment by svlandeg** [comment]
Hi!

In fact in spaCy, there is only one "class" for English as well; `en`, which attempts to cover both American and English grammar. Likewise, the aim of the `pt` model is to cover both versions of Portuguese. Some related discussion can be found here: https://github.com/explosion/spaCy/pull/2759


> Brazilian and Portuguese grammars are the same so in theory, we don't need to have two separate languages. The differences would be in the meaning of some words. The changes that I made are valid for both.
